README.md

```diff
@@ -68,7 +66,7 @@ the enclave once the host is re-started.
 - [Documentation](https://docs.conclave.net)
   - We recommend to read the [Concepts chapter](https://docs.conclave.net/enclaves.html) to learn more about confidential computing, enclaves, and Conclave's architecture
 - [Video tutorials and bootcamps](https://www.youtube.com/channel/UCSZbii_Z5QXci6Sr-CvStuw)
-  -  We recommend to watch the video ["Getting started with Conclave in 5 minutes"](https://youtu.be/XGz_utvu6HE)
+  -  We recommend to watch the video ["Getting started with Conclave SDK 1.3"](https://youtu.be/nwyGL5OemSU)
```

Comment:
Instead of "We recommend **to** watch the video", it should be "We recommend watch**ing** the video".

Reply:
pushed in the change.